docs/resources/cci_namespace.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Cloud Container Instance (CCI)"
+---
+
+# huaweicloud_cci_namespace
+
+Manages a CCI namespace resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "namespace_name" {}
+
+resource "huaweicloud_cci_namespace" "test" {
+  name               = var.namespace_name
+  flavor             = "gpu-accelerated"
+  pool_size          = 15
```

Comment:
Currently, advanced settings are available only in region CN North-Beijing4.
so, I think you can delete them in a basic exapmle usage.

Reply:
The example has been updated, and the test has split into basic test and high-level test (container network).